docs/core/diagnostics/microsoft-diagnostics-netcore-client.md

```diff
@@ -73,21 +73,29 @@ public EventPipeSession StartEventPipeSession(
     IEnumerable<EventPipeProvider> providers,
     bool requestRundown = true,
     int circularBufferMB = 256);
+public Task<EventPipeSession> StartEventPipeSessionAsync(
+    IEnumerable<EventPipeProvider> providers,
+    bool requestRundown,
+    int circularBufferMB = 256,
+    CancellationToken token = default);
 ```
 
 Starts an EventPipe tracing session using the given providers and settings.
 
 * `providers` : An `IEnumerable` of [`EventPipeProvider`](#eventpipeprovider-class)s to start tracing.
 * `requestRundown`: A `bool` specifying whether rundown provider events from the target app's runtime should be requested.
 * `circularBufferMB`: An `int` specifying the total size of circular buffer used by the target app's runtime on collecting events.
+* `token` (for the Async overload): The token to monitor for cancellation requests.
 
 ```csharp
-public EventPipeSession StartEventPipeSession(EventPipeProvider providers, bool requestRundown=true, int circularBufferMB=256)
+public EventPipeSession StartEventPipeSession(EventPipeProvider provider, bool requestRundown = true, int circularBufferMB = 256)
```

Comment:
The coding style here is inconsistent throughout the file with regard to spaces around the ` = ` and also the wrapping style. Not sure which is "correct", but from a practical point of view, wrapping means more vertical scrolling and not wrapping means more horizontal scrolling. Horizontal scrolling seems more annoying, since you have to scroll the vertically anyway! I've kept the wrapping style inconsistency for now.

Reply:
That is a great point, thanks for the feedback. I agree with what you said, let's see if others have an opinion.